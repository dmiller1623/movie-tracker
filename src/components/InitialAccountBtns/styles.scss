@import "variables.scss";

.initialBtns {
  color: $off-white;
  text-decoration: none;
  margin: 0 .5rem;
  &:hover {
    color: $light-grey;
  }
}

span {
  color: $off-white;
}

.selected {
  color: $light-grey;
}
