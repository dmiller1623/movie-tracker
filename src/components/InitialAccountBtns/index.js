import React from 'react';
import { NavLink } from 'react-router-dom';

import './styles.css';

export const InitialAccountBtns = () => {
  return (
    <div>
      <NavLink activeClassName='selected' className='initialBtns' to='/login'>
        Login
      </NavLink>
      <span> | </span>
      <NavLink activeClassName='selected' className='initialBtns' to='/signup'>
        Sign Up
      </NavLink>
    </div>
  );
};
