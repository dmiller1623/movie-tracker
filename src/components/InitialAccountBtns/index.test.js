import React from 'react';
import { shallow } from 'enzyme';
import { InitialAccountBtns } from '../InitialAccountBtns';
import { BrowserRouter }  from 'react-router-dom';

describe('InitialAccountBtns', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(
      <BrowserRouter>
        <InitialAccountBtns />
      </BrowserRouter>);

    expect(wrapper.html()).toMatchSnapshot();
  });

});