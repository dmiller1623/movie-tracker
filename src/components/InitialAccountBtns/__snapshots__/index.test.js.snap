// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InitialAccountBtns should match the snapshot 1`] = `"<div><a href=\\"/login\\">Login</a><a href=\\"/signup\\">Sign Up</a></div>"`;
